common/authen.js

```diff
@@ -189,6 +195,7 @@
                     // user not logged in
                     $window.location = "/chaise/logout";
                 });
+                // TODO: Conclude chain w/ a .catch for catchAll
```

Comment:
Move to ermrestjs. Error cb needs to find what kind of error it is.
Set default logout url if undefined in chaiseConfig
- Not found = no session already logged out. Treat like success. Go to logout url.
- Other errors = rethrow error.